src/main/java/org/zalando/nakadi/service/EventPublisher.java

```diff
@@ -145,6 +151,11 @@ private void validateSchema(final JSONObject event, final EventType eventType) t
         }
     }
 
+    private void validateEventSize(final BatchItem item) throws EventValidationException {
+        if (item.getEvent().toString().getBytes().length > nakadiSettings.getEventMaxBytes())
+            throw new EventValidationException("Event too large");
```

Comment:
@rcillo Pretty sure I removed the whole method :). The size computation is now in the `BatchItem` constructor